### PR TITLE
Complete quickstart guide

### PR DESCRIPTION
**Pull request by jschoone**

### CODE REVIEW

**Review by maxwolfs — APPROVED** [review]
lgtm!

File: doc/quickstart.md [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
 
-## Dependencies
+## Requirements
 
 - make
 - kubectl
-- clouds.yaml
-- environment.tfvars
+- terraform
+- yq v2
```
Comment (chess-knight) [inline doc/quickstart.md:8]: For me, it works only with v4.

File: doc/quickstart.md [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
 
-## Steps
+## Prepare the environment
 
-- `make create`
-- `make kubeconfig`
-- `kubectl --kubeconfig testconfig.kubeconfig get nodes`
-- `make clean`
+You need access to an OpenStack cluster.
```
Comment (chess-knight) [inline doc/quickstart.md:12]:
```suggestion
You need access to an OpenStack project.
```

File: doc/quickstart.md [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
+
+# Create your environment. This includes a management node as virtual machine
+# in your OpenStack environment as well as a Kubernetes testcluster.
+make create
+
+# Get the kubeconfig of the testcluster
+make get-kubeconfig
+
+# Interact with the testcluster
+kubectl --kubeconfig testcluster.kubeconfig get nodes
```
Comment (chess-knight) [inline doc/quickstart.md:38]:
```suggestion
kubectl --kubeconfig testcluster.yaml.<YOURCLOUD> get nodes
```

File: doc/quickstart.md [hunk 4]
```diff
@@ -1,22 +1,47 @@
...
+```
+
+## Teardown
+
+```
+make clean
```
Comment (chess-knight) [inline doc/quickstart.md:44]: Maybe add a link to teardown.md

It usually ends with an error and fullclean is needed

File: doc/quickstart.md (hunk 1 above)
Comment (garloff) [reply doc/quickstart.md:8]: Maybe add a note that there are two variants of `yq` which are in principle both supported, but the version based detection needing improvement?

File: doc/quickstart.md (hunk 4 above)
Comment (garloff) [reply doc/quickstart.md:44]: Hmmm, the standard quick-start scenario where you create just the testcluster should normally work with the standard make clean.

File: doc/quickstart.md (hunk 1 above)
Comment (jschoone) [reply doc/quickstart.md:8]: Good hint. Pip installed `yq 3.1.1` and I just thought that's the newest so I downgraded to the latest `v2` and it worked

File: doc/quickstart.md (hunk 4 above)
Comment (chess-knight) [reply doc/quickstart.md:44]: Sorry, you are right it works. I just messed up something.

**Pull request merged**